wordpress-mobile/WordPress-Login-Flow-Android
# Jetpack Focus: enable self-hosted login on the Jetpack app

This PR enables self-hosted site sign in for Jetpack app by removing the JETPACK_LOGIN_ONLY login mode. Jetpack will now behave identical to WordPress.

Parent PR - [Jetpack Focus: Enable self-hosted login on the Jetpack app](https://github.com/wordpress-mobile/WordPress-Android/pull/17094)

Testing Instructions
- See parent [PR](https://github.com/wordpress-mobile/WordPress-Android/pull/17094) for instructions
